CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Change log
 
-
+- [#528](https://github.com/mobilityhouse/ocpp/issues/528) Optional fields in datatypes require the default of None
```

Comment:
A changelog should answer the question: "does any of these changes might effect me?" 
Can you make this line in the Changelog a bit more explicit and mention the type you've changed?

Reply:
change to - v2.0.1 CertificateHashDataChainType childCertificateHashData requires the default of None